### PR TITLE
Update README and do some cleanup

### PR DESCRIPTION
**Pull request by lyeyixian**

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Frontend
+
+## Development
+
+1. Refer to `.env.example` file to create your own `.env` file
```
Comment (Ziyang-98) [inline frontend/README.md:5]: is there a .env.example in the frontend folder?

**Review by Ziyang-98 — COMMENTED** [review]
Might need to update the package-lock.json for frontend. Can do a npm install locally to sync the package-lock.json and package.json

File: frontend/README.md (hunk 1 above)
Comment (lyeyixian) [reply frontend/README.md:5]: Oh yea, there isn't I not sure what env is needed in the frontend can you help me add it?

Comment (Ziyang-98) [reply frontend/README.md:5]: yep I can add this

**Comment by lyeyixian** [comment]
> Might need to update the package-lock.json for frontend. Can do a npm install locally to sync the package-lock.json and package.json

I did that but the package-lock.json file doesn't change tho.

**Comment by Ziyang-98** [comment]
try deleting package-lock then do npm install

**Comment by lyeyixian** [comment]
Yup, tried that as well. The package-lock.json file remain the same.

**Review by Ziyang-98 — APPROVED** [review]
LGTM

**Pull request merged**